service/framework.go

```diff
@@ -462,6 +475,13 @@ func NewVersionedResources(config Config, k8sClient kubernetes.Interface, awsCon
 		}
 	}
 
+	// As 2.1.0 based on the same subset of resources as 1.0.1 - it just copied
+	// into new minor version.
+	var resourcesV2_1_0 []framework.Resource
```

Comment:
I have doubts about this, as we adding here new parameters, but in fact, do not change any resource in the bundle. Is it worth to make this cascade copying of all resources because of those flags?

Reply:
Yes, this is going to need changes. We need a new s3object resource that uses cloudconfigv4.

See the structure here. https://github.com/giantswarm/aws-operator/tree/master/service/awsconfig/v3

CC @kopiczko as you're also working on this.